internal/e2e/rollup/foo.js

```diff
@@ -6,6 +6,18 @@ import {name} from './bar';
 
 console.log(`${hello}, ${name} in ${fum}`);
 
+// Tests for @PURE annotations
+/*@__PURE__*/
+console.log('side-effect');
+
+class Impure {
+  constructor() {
+    console.log('side-effect')
+  }
+}
+
+/*@__PURE__*/new Impure();
```

Comment:
No spec to verify that the bundle is as expected?

Reply:
The spec is the golden files - they verify that these added lines didn't get retained